.NET/Microsoft.Recognizers.DataTypes.DataDrivenTests/TestTimex.cs

```diff
@@ -9,19 +9,19 @@ namespace Microsoft.Recognizers.DataTypes.DateTime.Tests
     public class TestTimex
     {
         [TestMethod]
-        public void Timex_FromDate()
+        public void DataTypes_Timex_FromDate()
         {
             Assert.AreEqual("2017-12-05", Timex.FromDate(new System.DateTime(2017, 12, 5)).TimexValue);
```

Comment:
Let's also rename the Timex class to TimexExpression or Expression?
I know Microsoft.Recognizers.Text.DataTypes.TimexExpression.TimexExpression is probably not the greatest class name, but I don't have a better suggestion. :)